tests/users/test_data.py

```diff
@@ -188,37 +198,41 @@ async def test_administrator(
         )
 
     async def test_force_reset(
-        self,
-        data_layer: DataLayer,
-        fake2: DataFaker,
-        mongo: Mongo,
+        self, data_layer: DataLayer, fake2: DataFaker, mongo: Mongo, pg: AsyncEngine
     ):
         """
         Test that setting and unsetting ``force_reset`` works as expected.
         """
         user = await fake2.users.create()
+        async with (AsyncSession(pg) as session):
+            row = await session.get(SQLUser, 1)
+            assert await get_one_field(mongo.users, "force_reset", user.id) is False
 
-        assert await get_one_field(mongo.users, "force_reset", user.id) is False
-
-        user = await data_layer.users.update(
-            user.id, UpdateUserRequest(force_reset=True)
-        )
-
-        assert user.force_reset is True
-        assert await get_one_field(mongo.users, "force_reset", user.id) is True
-
-        user = await data_layer.users.update(
-            user.id, UpdateUserRequest(force_reset=False)
-        )
-
-        assert user.force_reset is False
-        assert await get_one_field(mongo.users, "force_reset", user.id) is False
+            assert row.force_reset is False
+            assert user.force_reset is False
+            user = await data_layer.users.update(
+                user.id, UpdateUserRequest(force_reset=True)
+            )
```

Comment:
Please use whitespace to separate concepts to some extent. One line below a `with` block is a good place to start.
```suggestion
            )

```